src/Packetery/Module/Order/Metabox.php

```diff
@@ -188,6 +189,10 @@ public function register(): void {
 		add_action(
 			'admin_init',
 			function () {
+				if ( ! $this->detailCommonLogic->isPacketeryOrder() ) {
```

Comment:
Nejde ta podmínka dát na začátek metody register() ? Pokud objednávka není na Zásilkovnu, nemusíme oba hooky vůbec registrovat a v metodě add_metaboxes nemusíme znovu zjišťovat,  jestli je objednávka na zásilkovnu.

Reply:
Objednávka v době registrace ještě není z kontextu dostupná, proto je to zabalené do toho admin_init.